Client.xcodeproj/project.pbxproj

```diff
@@ -4564,6 +4579,7 @@
 				E68E7ADC1CAC208200FDCA76 /* SetupPasscodeViewController.swift in Sources */,
 				7B3631EA1C244FEE00D12AF9 /* Theme.swift in Sources */,
 				7BC68D321CC153920043562A /* AppMenuItem.swift in Sources */,
+				A9530A861CCF45FB00F01D0E /* UserScript.swift in Sources */,
```

Comment:
Can you be a bit more descriptive with the name of the user script file? e.g. `NoImageMode.js`